test/e2e/app-dir/next-font.test.ts

```diff
@@ -0,0 +1,237 @@
+import { createNext, FileRef } from 'e2e-utils'
+import { NextInstance } from 'test/lib/next-modes/base'
+import { renderViaHTTP } from 'next-test-utils'
+import cheerio from 'cheerio'
+import path from 'path'
+import webdriver from 'next-webdriver'
+
+describe('app dir next-font', () => {
+  if ((global as any).isNextDeploy || (global as any).isNextDev) {
+    it('should skip next deploy for now', () => {})
+    return
+  }
+
+  if (process.env.NEXT_TEST_REACT_VERSION === '^17') {
+    it('should skip for react v17', () => {})
+    return
+  }
+  let next: NextInstance
+
+  beforeAll(async () => {
+    next = await createNext({
+      files: new FileRef(path.join(__dirname, 'next-font')),
+      dependencies: {
+        '@next/font': 'canary',
+        react: 'experimental',
+        'react-dom': 'experimental',
+      },
+      skipStart: true,
+    })
+    await next.start()
+  })
+  afterAll(() => next.destroy())
+
+  describe('import values', () => {
+    it('should have correct values at /', async () => {
+      const html = await renderViaHTTP(next.url, '/')
+      const $ = cheerio.load(html)
+
+      // layout
+      expect(JSON.parse($('#root-layout').text())).toEqual({
+        className: '__className_93eede',
+        variable: '__variable_93eede',
+        style: {
+          fontFamily: "'__font1_93eede'",
```

Comment:
Can we apply some normalizing on these values so the test doesn't need to be updated every time the hash changes?